Source/Core/VideoBackends/Vulkan/TextureCache.cpp

```diff
@@ -338,8 +340,28 @@ TextureCache::TCacheEntry::~TCacheEntry()
     g_command_buffer_mgr->DeferFramebufferDestruction(m_framebuffer);
 }
 
-void TextureCache::TCacheEntry::Load(const u8* buffer, unsigned int width, unsigned int height,
-                                     unsigned int expanded_width, unsigned int level)
+inline void CopyImageToStagingBuffer(u32 height, const u8* buffer, u8* upload_buffer,
```

Comment:
Whoops, left there from earlier code that's not used anymore and I forgot to remove it!